### ISSUE TITLE
New navigation unavailable after updating WooCommerce to 5.4

### ISSUE THREAD

**Issue by adrianduffell** [woocommerce/woocommerce-admin#7220]
**Describe the bug**

The old navigation is rendered when WooCommerce 5.4 is installed, despite previously opting-in to the new navigation from the settings.

This seems caused by the Core feature config being loaded, since at present core bundles the latest version of WC Admin and overrides the plugin version config.

**To Reproduce**

On a site with WooCommerce 5.3 and WC Admin 2.3 installed

1. Enable navigation in WooCommerce > Settings > Advanced > Features
2. See that new nav is rendered.
3. Update WooCommerce to 5.4
4. See that the old nav is rendered.

**Expected behavior**
As a user, I think I would expect the new navigation to continue to render when I've previously selected it, despite updating WooCommerce.

Perhaps the navigation could be conditionally enabled in the Core config using the `woocommerce_admin_features` filter, if users have previously selected it.

**Comment by adrianduffell** [comment]
This will be resolved temporally once the next release of WC Admin (2.4) is available, however will then occur again with the next release of WooCommerce.

**Comment by adrianduffell** [comment]
Not stale

**Comment by mattsherman** [comment]
The heart of this issue is that Navigation isn't available in Core (using the bundled WCA). It would seem to me that we should make Navigation available in Core (disabled by default). Does anybody disagree with that?

/cc @pmcpinto

**Comment by pmcpinto** [comment]
>  It would seem to me that we should make Navigation available in Core (disabled by default). 

I think this is going to happen with the monorepo unification project, right?

**Comment by mattsherman** [comment]
> > It would seem to me that we should make Navigation available in Core (disabled by default).
> 
> I think this is going to happen with the monorepo unification project, right?

Not necessarily automatically.

We should make a clear decision on how we want to handle the Navigation feature and then we can make sure that the implementation supports that decision.

**Comment by pmcpinto** [comment]
I'm sorry if I wasn't clear, let me rephrase it: once wc admin is merged, the best way to show an option to enable experimental features is directly through core, correct? Unless we create other plugins. 

> We should make a clear decision on how we want to handle the Navigation feature and then we can make sure that the implementation supports that decision.

The future of the new nav roll-out will be defined once we have more news about the annual plans. But making the nav disabled by default as you proposed seems to be a low-risk decision that can solve this issue. I'm assuming that this is impacting eComm users with the nav enabled.

**Comment by adrianduffell** [comment]
I think we can now close this out. 

As part of the merge into Core, the navigation feature has been promoted to the Core release in https://github.com/woocommerce/woocommerce-admin/issues/8298 

@pmcpinto  This aligns with your comment above:

> the best way to show an option to enable experimental features is directly through core, correct? Unless we create other plugins.
